CONFIGURING_CLION.md

```diff
@@ -0,0 +1,24 @@
+# Configuring Clion for Rust-only development
+These are the steps to build the application-services desktop in clion:
+This works on Ubuntu 18.04 and OSx10.0, but will probably work on anything.
```

Comment:
s/OSx10.0/OSX 10.15

I'm assuming this was Catalina? 10.0 is a very old release.

Reply:
Mojave 10.14.6 sorry